src/renderers/shared/fiber/__tests__/ReactIncremental-test.js

```diff
@@ -2184,4 +2184,87 @@ describe('ReactIncremental', () => {
       'componentDidUpdate',
     ]);
   });
+
+  it('should reuse memoized work after an interrupted low-priority render', () => {
```

Comment:
Let's rename the test so it's clear that this is specifically about updating the pointers before calling lifecycles.